src/modules/graphics/graphics.c

```diff
@@ -553,6 +553,52 @@ void lovrGraphicsSetProjection(mat4 projection) {
   state.frameDataDirty = true;
 }
 
+void lovrGraphicsMatrixGetTransform(mat4 dst, int* count, MatrixRequest request) {
+  *count = lovrGraphicsGetIsStereo() ? 2 : 1;
+  const size_t matrixSize = 16*sizeof(float);
+  for(int eye = 0; eye < *count; eye++) {
+    mat4 eyeMatrix = dst + 16*eye;
+    switch (request) {
+      case MATRIX_REQUEST_MODEL:
+        memcpy(eyeMatrix, state.transforms[state.transform], matrixSize);
```

Comment:
Note: This will return 2 copies of the model matrix even though the model transform will not (can not) differ between the two eyes.